doxygen/01_Installation.dox

```diff
@@ -10,33 +10,33 @@ Point your browser to \a https://github.com/genn-team/genn/releases
 and download a release from the list by clicking the relevant source
 code button. Note that GeNN is only distributed in the form of source
 code due to its code generation design. Binary distributions would not
-make sense in this framework and are not provided. 
+make sense in this framework and hence are not provided. 
 After downloading continue to install GeNN as described in the \ref installing section below.
 
 \section GitSnapshot Obtaining a Git snapshot
 
-If it is not yet installed on your system, download and install Git
-(\a http://git-scm.com/). Then clone the GeNN repository from Github
+If it is not yet installed on your system, download and install <a href="http://git-scm.com/">Git<\a>.
+Then clone the GeNN repository from Github
 \code
 git clone https://github.com/genn-team/genn.git
 \endcode
 The github url of GeNN in the command above can be copied from the
-HTTPS clone URL displayed on the GeNN Github page (\a https://github.com/genn-team/genn).
+HTTPS clone URL displayed on the GeNN <a href="https://github.com/genn-team/genn">Github page<\a>.
 
 This will clone the entire repository, including all open branches.
 By default git will check out the master branch which contains the
 source version upon which the next release will be based. There are other 
 branches in the repository that are used for specific development 
 purposes and are opened and closed without warning.
 
-As an alternative to using git you can also download the full content of
-GeNN sources clicking on the "Download ZIP" button on the bottom right
-of the GeNN Github page (\a https://github.com/genn-team/genn).
```

Comment:
When processed through doxygen these links work fine as is - so no need to make explicit hyperlinks.